docker/Dockerfile

```diff
@@ -7,11 +7,13 @@ COPY ./src/ ./src/
 COPY ./assets/ ./assets/
 COPY ./locales/ ./locales/
 
-# Rust is needed for orjson
-RUN apk add --no-cache git python3 py3-pip rust cargo tini && \
+RUN apk add --no-cache git python3 py3-setuptools tini && \
     addgroup -g 1000 -S priviblur && \
     adduser -u 1000 -S priviblur -G priviblur && \
-    pip3 install --break-system-packages -r requirements.txt && \
+    apk add --no-cache py3-pip && \
```

Comment:
Why move this here instead of the first line?

Reply:
It looks like a mistake honestly :')

Also, you should have added a comment to say that it's only a build dependency